priv/repo/migrations/20190406081526_change_podcast_add_network_id.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Radiator.Repo.Migrations.ChangePodcastAddNetworkId do
+  use Ecto.Migration
+
+  def change do
+    alter table(:podcasts) do
+      add :network_id, references(:networks, on_delete: :nothing)
```

Comment:
Yes, I do prefer editing existing migrations at this stage in the project to keep them clean / expressive / readable.

Reply:
okay, so this will be gone and create networks be moved at position 1 for migrations so e..g this refernce can be specified in the create podcasts migration?

Reply:
right, will change it accordingly